### ISSUE TITLE
Its an interesting work, but the comparison seems a little unfair

### ISSUE THREAD

**Issue by liuweijie19980216** [prannaykaul/lvc#4]
I noticed that KNN is trained using the pre-trained VIT model, which means that the work also uses training samples of pre-trained VIT, which seems unfair.

**Comment by prannaykaul** [comment]
Hello, we use a ViT pretrained with the self-supervised DINO method. This uses ImageNet-1k but **without** labels. So we do not use any extra labelled data.

**Comment by liuweijie19980216** [comment]
Thanks for your reply. Actually, I love the work very much.
